react/build.boot

```diff
@@ -52,15 +54,36 @@
                                           (:tree fileset)))]
         (handler fileset)))))
 
-(defn package-part [{:keys [extern-name namespace project dependencies requires]}]
+(deftask surrogate
+  "Creates an empty surrogate js fies"
+  [f filename FILENAME str "File name"
+   p project  PROJECT  sym "Project"]
+  (let [tmp    (tmp-dir!)
+        target (io/file tmp filename)]
+    (with-pre-wrap fileset
+      (io/make-parents target)
+      (spit target (str "// Surrogate package for "
+                        (str/replace (name project) #"-surrogate$" "")))
+      (-> fileset
+          (add-resource tmp)
+          commit!))))
+
+(defn package-part [{:keys [extern-name namespace project dependencies requires surrogate?]}]
   (with-files (fn [x] (= extern-name (.getName (tmp-file x))))
     (comp
-      (download :url (format "https://unpkg.com/%s@%s/dist/%s.js" (npm-project project) +lib-version+ (name project))
-                :checksum (:dev (get checksums project)))
-      (download :url (format "https://unpkg.com/%s@%s/dist/%s.min.js" (npm-project project) +lib-version+ (name project))
-                :checksum (:min (get checksums project)))
-      (sift :move {(re-pattern (format "^%s.js$" (name project)))     (format "cljsjs/%1$s/development/%1$s.inc.js" (name project))
-                   (re-pattern (format "^%s.min.js$" (name project))) (format "cljsjs/%1$s/production/%1$s.min.inc.js" (name project))})
+     (if surrogate?
+       (comp
+        (surrogate :filename (format "cljsjs/%1$s/development/%1$s.inc.js" (name project))
+                   :project project)
+        (surrogate :filename (format "cljsjs/%1$s/production/%1$s.min.inc.js" (name project))
```

Comment:
So if `:file-min` is not set in deps.cljs, `:file` is used automatically? The reason I left this in was because I wasn't clear on the mechanism by which the minified file is selected.